### ISSUE TITLE
Introduce proper motion UI flow disables / loading states

### ISSUE THREAD

**Issue by rdig** [JoinColony/colonyCDapp#1803]
The current motion UI works perfectly when tested locally, however, in a production environment where the wait and load times are way longer, it starts to be unreliable as there are wait states that are not accounted for:
- loading spinners to tell the user "stuff" is happening in the background
- button disables
- button loading states
- components that still render after they are no longer needed

The list of things that need to be introduced / accounted for is:
_To be added_

**Comment by melyndav** [comment]
Hi @rdig thanks for tracking the issue. Before anyone picks up this issue, I'd like for this to be planned first and a loading approach decided upon (i.e all content showing as skeletons). Please hold off until we do up a spec for this.

**Comment by rdig** [comment]
@melyndav  This is not ready to start work on yet. As you can see from my "To be added" section, investigation is still needed to track the various states that actually need to be added in, in the various motion flow steps.

Once that is done, we can discuss about how you want to UI to look like.

**Comment by melyndav** [comment]
@rdig sorry, I saw it tagged as 'to do' in cycle #184 which I thought gave the green light for someone to pick it up which threw me off. 

<img width="335" alt="image" src="https://github.com/JoinColony/colonyCDapp/assets/82127904/9d0595d1-f6f6-4149-b530-7bdac0f130a2">

**Comment by rdig** [comment]
Closing this as it's grossly out of date, most of the issues that were intended here were tracked and fixes by individual issues
